geist/200.css

```diff
@@ -0,0 +1,8 @@
+/* geist-latin-200-normal */
+@font-face {
+  font-family: 'Geist Sans';
+  font-style: normal;
+  font-display: swap;
+  font-weight: 200;
+  src: url(./files/geist-latin-200-normal.woff2) format('woff2'), url(./files/geist-latin-200-normal.woff) format('woff');
```

Comment:
Fontsource occasionally regenerates the whole repository from scratch and the custom packager relies on the font file names to derive the name of the package. We need to rename the font files for this to not cause any future issues.

Reply:
Should the naming follow this pattern?
 `geist-sans-latin-[weight]-normal.woff2`

Reply:
Yup! That would be perfect.

Reply:
@ayuhito Updated.

Reply:
Sorry for not noticing this sooner. I think we need to also rename the folder to geist-sans 😓 Hopefully that will the last thing to address.

Reply:
No worries @ayuhito, I should have spotted it. Can't wait to see the package!

Reply:
Unfortunately, I just realised we are missing another thing 😓 The `geist-sans` folder is in the root of the monorepo. Can you move the directory into `fonts/other`?

Reply:
@ayuhito Moved.